packages/core/src/codecatalyst/activation.ts

```diff
@@ -37,6 +37,15 @@ export async function activate(ctx: ExtContext): Promise<void> {
 
     await authProvider.restore()
 
+    // if connection is shared with CodeWhisperer, check if CodeCatalyst scopes are expired
+    if (authProvider.activeConnection && authProvider.isSharedConn()) {
+        try {
+            await createClient(authProvider.activeConnection, undefined, undefined, undefined, {
+                showReauthPrompt: false,
+            })
+        } catch {}
```

Comment:
why is the exception swallowed without logging? if it is normal for createClient to throw an exception, there should be a code comment explaining why we don't want to handle (or even log, apparently) the exception.

Reply:
An exception is expected, but you're right, there should be a log. I'll add one